components/script/dom/element.rs

```diff
@@ -1929,17 +1929,17 @@ impl ElementMethods for Element {
 
     // https://dom.spec.whatwg.org/#dom-parentnode-firstelementchild
     fn GetFirstElementChild(&self) -> Option<Root<Element>> {
-        self.upcast::<Node>().child_elements().next()
+        self.upcast::<Node>().children::<Element>().next()
     }
 
     // https://dom.spec.whatwg.org/#dom-parentnode-lastelementchild
     fn GetLastElementChild(&self) -> Option<Root<Element>> {
-        self.upcast::<Node>().rev_children().filter_map(Root::downcast::<Element>).next()
+        self.upcast::<Node>().rev_children::<Element>().filter_map(Root::downcast::<Element>).next()
```

Comment:
Is there a better way to deal with this?  Same behavior in a few other places as well.
```
1937 |         self.upcast::<Node>().rev_children().filter_map(Root::downcast::<Element>).next()
     |                               ^^^^^^^^^^^^ cannot infer type for `T`
```